frontend/src/AlertsManagement/AlertsManagement.js

```diff
@@ -1,15 +1,97 @@
 import React, { Component } from 'react';
-import AlertsContent from './AlertsContent'
+import { Switch, Route } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import AlertsContent from './AlertsContent';
+import AlertInfo from './AlertInfo';
 import NavBar from '../NavBar';
+import { tabLabels } from './management_constants';
+import { getAlertsData } from './management_helpers';
```

Comment:
ooc, is it a convention to name `helpers` and `constants` with lower case and underscore instead of camel case?

Reply:
honest answer: don't know. i mostly followed this because I named the time utility file `time_utils.js`. maybe Paula/Helen can comment on file names

Reply:
i think as long as the naming is consistent, it should be fine!

but also interested to know what Paula/Helen thinks :)